src/components/common/SortableWalletListRow.js

```diff
@@ -40,28 +40,24 @@ class SortableWalletListRow extends Component<Props, State> {
         {...this.props.sortHandlers}
       >
         {walletData.currencyCode ? (
-          <View style={[styles.rowContent, styles.sortableRowContent]}>
+          <View style={[styles.rowContent]}>
             <View style={[styles.rowDragArea]}>
-              <Image source={sort} style={{ height: 15, width: 15 }} />
+              <Image source={sort} style={styles.rowDragIcon} />
             </View>
-            <View style={[styles.rowNameTextWrap]}>
-              {Platform.OS === 'ios' && (
-                <View style={[styles.rowNameTextWrapIOS]}>
-                  <T style={[styles.rowNameText]} numberOfLines={1}>
-                    {symbolImageDarkMono && <Image style={[styles.rowCurrencyLogoIOS]} transform={[{ translateY: 3 }]} source={{ uri: symbolImageDarkMono }} />}{' '}
-                    {cutOffText(name, 34)}
-                  </T>
-                </View>
-              )}
-              {Platform.OS === 'android' && (
-                <View style={[styles.rowNameTextWrapAndroid]}>
-                  {symbolImageDarkMono && <Image style={[styles.rowCurrencyLogoAndroid]} source={{ uri: symbolImageDarkMono }} />}
-                  <T style={[styles.rowNameText]} numberOfLines={1}>
-                    {cutOffText(name, 34)}
-                  </T>
-                </View>
+            <View style={styles.rowIconWrap}>
+              {symbolImageDarkMono && (
+                <Image
+                  style={[styles.rowDragCurrencyLogo, { marginLeft: 0, marginRight: 5 }]}
+                  transform={[{ translateY: 3 }]}
+                  source={{ uri: symbolImageDarkMono }}
+                />
               )}
             </View>
+            <View style={[styles.rowNameTextWrapAndroidIos]}>
+              <T style={[styles.rowNameText]} numberOfLines={2} adjustsFontSizeToFit={true} minimumFontScale={0.6}>
+                {Platform.OS === 'ios' ? name : cutOffText(name, 34)}
```

Comment:
For Android please use ellipsesMode instead of just cutting off text. cutOffText knows nothing about the screen size and available space.

https://facebook.github.io/react-native/docs/text#ellipsizemode

Reply:
defaulted to what? If it's already on, then please just remove the cutOffText()

Reply:
It is needed for the ios platform. That is why there the conditional is for.

Also, it is exactly the same code on the unsorted list that we always use.

Reply:
How is it needed for iOS when the conditional above only has it running on Android? If it’s on the unsourced list code, we need to remove it there also. It was always a bad idea to hardcode ellipses to a specific char length.

Reply:
Let me check it out without that code and will evaluate if it cuts off the same as before

Reply:
Updated